packages/grafana-ui/src/utils/debug.ts

```diff
@@ -0,0 +1,30 @@
+import { Logger } from './logger';
+
+/**
+ * Allows debug helpers attachement to the window object
+ * @internal
+ */
+export function attachDebbuger(key: string, thebugger?: any, logger?: Logger) {
+  if (process.env.NODE_ENV !== 'production') {
+    let completeDebugger = thebugger ? { ...thebugger } : {};
+
+    if (logger !== undefined) {
+      completeDebugger = { ...completeDebugger, enable: () => logger.enable(), disable: () => logger.disable() };
+    }
+
+    // @ts-ignore
+    let debugGlobal = window['_debug'];
+    if (debugGlobal) {
+      debugGlobal = {
+        ...debugGlobal,
+        [key]: completeDebugger,
+      };
+    } else {
+      debugGlobal = {
+        [key]: completeDebugger,
+      };
+    }
+    // @ts-ignore
+    window['_debug'] = debugGlobal;
```

Comment:
nit: might be less complex to write it like this?
```suggestion
    let debugGlobal = window['_debug'] ?? {};
    debugGlobal[key] = completeDebugger;
    // @ts-ignore
    window['_debug'] = debugGlobal;
```